ci/scripts/unit.sh

```diff
@@ -3,5 +3,8 @@
 export GOPATH=$(pwd)/go
 
 pushd $GOPATH/src/github.com/ONSdigital/florence
-  make test
+make test
+  if  prettier --check "src/app/**/*.js" | tee /dev/stderr | grep -q "Code style issues found"; then
```

Comment:
just out of interest, I looked at this, and found on the [website](https://prettier.io/docs/en/cli.html#check):

> The command will return exit code 1 in the second case, which is helpful inside the CI pipelines

so, you could just use (instead of these three lines):

```
prettier --check "src/app/**/*.js" || exit $?
```

Reply:
thinking about it, you might want to put that test into the `Makefile` instead of this file:

```
test-pretty:
            prettier --check "src/app/**/*.js"
```

and consider adding that target to the `test` target in there, too:

`test: test-go test-npm test-pretty`